woocommerce/woocommerce
# [Experimental] Product Filters Redesign > Parent block: Enable block customizations

### Changes proposed in this Pull Request:

This PR adds support to some customizations for the Product Filters Wrapper block. Merchants should be able to edit:

- Border
- Text Color
- Background Color

In the future, we will work on other issues, including support for dimensions and new color settings for the overlay text, icons, etc.





Closes #47037 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Navigate to Appearance > Editor > Templates > Product Catalog template - or any archive template.
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "Product Filters (Experimental)" in the block inserter or browse the available blocks until you find it. Add it to your content.
5. Select the "Product Filters (Experimental)" block.
6. In the Inspector Control, make sure you have controls to change the Text & Background Color, the Typography and the Border.

![CleanShot 2024-06-11 at 16 31 51@2x](https://github.com/woocommerce/woocommerce/assets/20469356/0e526c8e-c1b1-4978-a070-73c705aac5d8)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
Enable block customizations for the Product Filters block

</details>
